options/posix/include/pthread.h

```diff
@@ -50,6 +50,8 @@ extern "C" {
 #define PTHREAD_MUTEX_INITIALIZER {0, 0, 0}
 #define PTHREAD_RWLOCK_INITIALIZER {0, 0, 0}
 
+#define SIGALL_SET ((sigset_t *)(const unsigned long long [2]){ (unsigned long)-1, (unsigned long)-1 })
```

Comment:
This macro is broken, mlibc's ``sigset_t`` is of type ``long`` not ``uint64_t[2]``